### PR TITLE
DIA-207 Adding tracking for genes clicks and Highlights on the artist page

### PR DESCRIPTION
**Pull request by leodmz**
The type of this PR is: Enhancement

This PR resolves [DIA-207]

### Description

This PR adds tracking in Cohesion for both Eigen and Force for 2 events:
- click on one of the gens in the "related categories" section in the about tab of an artist page
- click on one of the accordion in Highlight and Achievements in the Artist header

[DIA-207]: https://artsyproduct.atlassian.net/browse/DIA-207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Schema/Events/Click.ts [hunk 1]
```diff
@@ -1742,4 +1742,61 @@ export interface ClickedVerifiedRepresentative {
...
+ *    destination_screen_owner_type: "Gene"
+ *    destination_screen_owner_id: "58de681f275b2464fcdde097"
+ *    subject: ""
+ *  }
+ * ```
+ */
+export interface ClickedRelatedCategory {
+  action: ActionType.clickedRelatedCategory
+  context_module: ContextModule
+  context_screen_owner_type: PageOwnerType
```
Comment (araujobarret) [inline src/Schema/Events/Click.ts:1769]: Aren't these supposed to be `page` instead of `screen` fields for the web and `screen` when comes to mobile?

Comment (araujobarret) [reply src/Schema/Events/Click.ts:1769]: Tap events are fine as far as I checked, they use `screen` fields

File: src/Schema/Events/Click.ts [hunk 2]
```diff
@@ -1742,4 +1742,61 @@ export interface ClickedVerifiedRepresentative {
...
+ *    context_screen_owner_type: "Artwork",
+ *    context_screen_owner_id: "58de681f275b2464fcdde097",
+ *    context_screen_owner_slug: "damien-hirst",
+ *    destination_screen_owner_type: "Gene"
+ *    destination_screen_owner_id: "58de681f275b2464fcdde097"
+ *    subject: ""
+ *  }
+ * ```
+ */
+export interface ClickedRelatedCategory {
```
Comment (araujobarret) [inline src/Schema/Events/Click.ts:1766]: We use the word `gene` everywhere, so isn't better to use the same naming for the events too to keep the terminology consistent?

File: src/Schema/Events/Click.ts (hunk 1 above)
Comment (leodmz) [reply src/Schema/Events/Click.ts:1769]: you're right, it's a copy paste from app that I forgot to amend

File: src/Schema/Events/Click.ts (hunk 2 above)
Comment (leodmz) [reply src/Schema/Events/Click.ts:1766]: Good call, I'll fix that

File: src/Schema/Events/Click.ts [hunk 3]
```diff
@@ -1742,4 +1742,61 @@ export interface ClickedVerifiedRepresentative {
...
+ * A user clicks one of the related categories(genes) in the artist about tab
+ *
+ *  This schema describes events sent to Segment from [[clickedGene]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedGene",
+ *    context_module : "Young British Artists",
+ *    context_screen_owner_type: "Artwork",
```
Comment (araujobarret) [inline src/Schema/Events/Click.ts:1757]: change the example too, please 🙏

Comment (leodmz) [reply src/Schema/Events/Click.ts]: done

**Review by araujobarret — APPROVED** [review]
LGTM!

**Comment by artsyit** [comment]
:rocket: PR was released in [`v4.153.0`](https://github.com/artsy/cohesion/releases/tag/v4.153.0) :rocket:

**Pull request merged**